test/check_dbmail_message.c

```diff
@@ -834,7 +813,7 @@ START_TEST(test_dbmail_message_construct)
 	const gchar *sender = "foo@bar.org";
 	const gchar *subject = "Some test";
 	const gchar *recipient = "<bar@foo.org> Bar";
-	gchar *body = g_strdup("\ntesting\n\n�����\n\n");
+	gchar *body = g_strdup("\ntesting\n\n�����\n\n");
```

Comment:
something went wrong with the encoding of the file, it seems

Reply:
These two changes were accidental, and I've reverted the lines to what it was in commit e6e1eea.

I've checked all other commits, but I did not see any other accidental changes.